### DISCUSSION TITLE
Structuring the application

### DISCUSSION THREAD

**Discussion by sasachan** [NeoSOFT-Technologies/rest-dot-net-core#33] [Show and tell]
Use Clean Architecture solution, each project has clear responsibilities. As such, certain types belong in each project and you’ll frequently find folders corresponding to these types in the appropriate project.

https://github.com/jasontaylordev/CleanArchitecture
https://github.com/ardalis/CleanArchitecture
https://github.com/iammukeshm/CleanArchitecture.WebApi
https://github.com/ivanpaulovich/clean-architecture-manga
https://github.com/Amitpnk/Clean-Architecture-ASP.NET-Core
https://github.com/jasontaylordev/NorthwindTraders
https://github.com/thangchung/clean-code-dotnet

**Comment by sthitadhi-chakraborty-neosoft** [comment]
Repository pattern, solid principle design patterns can be followed
